source/common/res/features/hide-help/main.js

```diff
@@ -3,9 +3,19 @@
   if (typeof ynabToolKit !== 'undefined' && ynabToolKit.pageReady === true) {
     ynabToolKit.hideHelp = (function () {
       return {
+        hideHelp: 'true',
```

Comment:
if you use the second parameter of `getToolkitStorageKey`, you can use `true`/`false` primitives here.